### PR TITLE
[feat] Listing Products on PLP

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- This PR contains Product Listing Page and fetching products on first render from API.
- Please ignore the Image Alignment for the card. will fix that in the upcoming PR's.

### CODE REVIEW

File: src/Molecules/CSS/productcard.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* .ecom-card {
```
Comment (AnuragSahu11) [inline src/Molecules/CSS/productcard.css:1]: remove comments

File: src/Pages/ProductListing.jsx [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+
+
+    useEffect(() => {
+      (async () => {
+        setLoading(true);
+        setError();
+
+        try {
+            const resp = await axios.get('/api/products');
+            console.log(resp.data.products);
```
Comment (AnuragSahu11) [inline src/Pages/ProductListing.jsx:22]: you can remove console.log

**Review by AnuragSahu11 — APPROVED** [review]
You can remove comments and console.log other than that good use of try, catch and self explanatory variable names, Good to merge.

File: src/Pages/ProductListing.jsx (hunk 2 above)
Comment (MaruthiPothuganti) [reply src/Pages/ProductListing.jsx:22]: Thankyou., will clenup.

File: src/Hooks/useAxios.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+axios.defaults.baseURL = "api/";
+
+export const useAxios = () => {
+    const [response, setResponse] = useState(null);
+    const [error, setError] = useState();
+    const [loading, setLoading] = useState(false);
+
+    const fireRequest = async (params) => {
+         setError();
```
Comment (Hamdrive) [inline src/Hooks/useAxios.jsx:12]: consider setting to either false/""/null instead of just empty ()
makes it easier to understand state is being reset

**Review by Hamdrive — COMMENTED** [review]
Good implementation of axios custom hook, esp. with try/catch/finally block
Overall nice work done with logic implemented, lots more to do

Also look into converting your components into further smaller components, following React logic and hierarchy
You can cleanup and remove unwanted files from Mockbee/CRA boilerplate

**Pull request merged**